sdks/python/apache_beam/pipeline.py

```diff
@@ -345,8 +345,9 @@ def from_runner_api(proto, runner, options):
     p = Pipeline(runner=runner, options=options)
     from apache_beam.runners import pipeline_context
     context = pipeline_context.PipelineContext(proto.components)
+    root_transform_id, = proto.root_transform_ids
```

Comment:
To ensure there's only one. (If I indexed, I might silently drop any others. Whether we keep this as a list or go back to a singleton is TBD.)